README.rst

```diff
@@ -1307,6 +1307,10 @@ Choose from the list of available rules:
 
   Instructions must be terminated with a semicolon.
 
+* **semicolon_on_new_line_for_chained_call**
```

Comment:
This rule is conflicting with `no_multiline_whitespace_before_semicolons` (one rule will put `;` to new line, other will put it back to same line). This is not acceptable.

What you need is:
- create new fixer that will have configuration option, which would allow to configure fixer to move `;` to new line or same line
- deprecate `no_multiline_whitespace_before_semicolons`
- make `no_multiline_whitespace_before_semicolons` proxy fixer to newly created fixer with needed configuration

Reply:
True, this would not be correct. I've created the new fixer `multiline_whitespace_before_semicolons` with a default option `no_multiline`, and deprecated the old one.